e2e/e2e_test.go

```diff
@@ -218,7 +218,7 @@ func TestUpgradesMajorVersion20to21(t *testing.T) {
 				current.Spec.Image.Name = "cockroachdb/cockroach:v21.1.0"
 				require.NoError(t, sb.Update(current))
 
-				RequireClusterToBeReadyEventuallyTimeout(t, sb, builder, 500*time.Second)
+				RequireClusterToBeReadyEventuallyTimeout(t, sb, builder, 600*time.Second)
```

Comment:
We need these backed out

Reply:
@chris it is already rolled back. See https://github.com/cockroachdb/cockroach-operator/blob/5b90ae2b235d3883d4532ada51a6c7aa6612e415/e2e/e2e_test.go#L221